src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/MalformedRoundaboutCheck.java

```diff
@@ -176,6 +201,19 @@ private boolean isExcludedHighway(final AtlasObject object)
                 HighwayTag.PEDESTRIAN, HighwayTag.FOOTWAY);
     }
 
+    /**
+     * Function for {@link SimpleEdgeWalker} that gathers connected edges that are part of a
+     * roundabout.
+     *
+     * @return {@link Function} for {@link SimpleEdgeWalker}
+     */
+    private Function<Edge, Stream<Edge>> isRoundaboutEdge()
+    {
+        return edge -> edge.connectedEdges().stream()
+                .filter(connected -> JunctionTag.isRoundabout(connected)
+                        && HighwayTag.isCarNavigableHighway(connected));
```

Comment:
Should we use the same constraints as validCheckForObject here? In that method we check `!isExcludedHighway` instead of `HighwayTag.isCarNavigableHighway()`

Reply:
@seancoulter @vladlemberg Resolved?